src/test/java/de/uhd/ifi/se/decision/management/jira/extraction/versioncontrol/TestGitRepositoryFSManager.java

```diff
@@ -1,278 +1,295 @@
-		String expectedContentPath = expectedBaseProjectUriBranchDir
-				+ File.separator
-				+ distinctFileInTempFolder;
-
-		assertEquals(1, findTemporaryDirectoryNames().length);
-
-		// test
-		testBranchFolderPreparation(branchName, expectedBaseProjectUriBranchDir, expectedContentPath);
-		// preparation should take the folder from temporary folders pool
-		assertEquals(0, findTemporaryDirectoryNames().length);
-
-		addDistinctFileInTempDir();
-		assertEquals(1, findTemporaryDirectoryNames().length);
-	}
-
-	@Test
-	public void testPrepareBranchDirectoryFromTempPrecedenceOverDefault() {
-		// setup
-		addDistinctFileInDefaultDir();
-		addDistinctFileInTempDir();
-		String expectedContentPath = expectedBaseProjectUriBranchDir
-				+ File.separator
-				+ distinctFileInTempFolder;
-
-		assertEquals(1, findTemporaryDirectoryNames().length);
-
-		// test
-		testBranchFolderPreparation(branchName, expectedBaseProjectUriBranchDir, expectedContentPath);
-		assertEquals(0, findTemporaryDirectoryNames().length);
-	}
-
-	@Test
-	public void testPrepareBranchDirectoryFromDefault() {
-		// setup
-		addDistinctFileInDefaultDir();
-		String expectedContentPath = expectedBaseProjectUriBranchDir
-				+ File.separator
-				+ distinctFileInDefaultFolder;
-
-		assertEquals(0, findTemporaryDirectoryNames().length);
-		// test
-		testBranchFolderPreparation(branchName, expectedBaseProjectUriBranchDir, expectedContentPath);
-		// preparation from default folder should not affect temporary folders
-		assertEquals(0, findTemporaryDirectoryNames().length);
-	}
+import javax.xml.bind.DatatypeConverter;
 
-	private void testBranchFolderPreparation(String branchName, String expectedDir, String expectedContentPath) {
-		String actualDir = FSmanager.prepareBranchDirectory(branchName);
-		assertNotNull(actualDir);
-		assertEquals(expectedDir, actualDir);
-		assertTrue(new File(expectedDir).isDirectory());
-		assertTrue(new File(expectedContentPath).isFile());
-	}
+import org.junit.Before;
+import org.junit.Test;
 
-	private static File getExampleDirectory() {
-		File directory = null;
-		try {
-			directory = File.createTempFile("clone", "");
-			directory.delete();
-			directory.mkdirs();
-		} catch (IOException e) {
-			e.printStackTrace();
-		}
-		return directory;
-	}
+public class TestGitRepositoryFSManager {
 
-	private static String getHash(String text) {
-		try {
-			MessageDigest md = MessageDigest.getInstance("MD5");
-			md.update(text.getBytes());
-			byte[] digest = md.digest();
-			return DatatypeConverter.printHexBinary(digest).toUpperCase().substring(0, 5);
-		} catch (NoSuchAlgorithmException e) {
-			e.printStackTrace();
-			return "";
-		}
+    private List<String> baseProjectUriDir;
+    private List<String> baseProjectUriDefaultDir;
+    private List<String> baseProjectUriTempDir;
+    private List<String> expectedBaseProjectUriBranchDir;
+    private List<String> repoUris;
+    private Map<String, String> folderForDefaultBranchNames;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'folderForDefaultBranchNames' could be replaced by a local variable.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=4916975)